pipelines/skylab/optimus/Optimus.wdl

```diff
@@ -44,6 +44,9 @@ workflow Optimus {
 
     # Set to true to count reads in stranded mode
     String use_strand_info = "false"
+    
+    # Set to true to count reads aligned to intronic regions in sn_rna mode
+    Boolean count_exons = false
```

Comment:
@khajoue2 Do we want to make it clear that this should NOT be run in sc mode?

Reply:
It will not harm the sc_rna mode anymore. We will skip making the second layer.